server/models/food.model.js

```diff
@@ -0,0 +1,39 @@
+const mongoose = require('mongoose');
+
+const FoodSchema = new mongoose.Schema({
```

Comment:
We dont need Food model

Reply:
We need this model, the relationship between food and user is n-n. We don't need to add the same food for each meal/tracker/user. Moreover, I processed all functions in controller, component etc with Food Schema